pkg/reconciler/reconcile.go

```diff
@@ -9,6 +9,36 @@ import (
 	"go.uber.org/zap"
 )
 
+type FleetReconciler struct {
+	// Data required for reconcile phases
+	// For AWS (concurrent safe) & IBM Clients decide whether they are created
+	// and destroyed per each reconcile or globals can be used
+}
+
+func (r *FleetReconciler) initReconciler(ctx context.Context) {
+	// Initializes the fields of FleetReconciler struct
+}
+
+func (r *FleetReconciler) reconcilePhases() (forman.Result, error) {
+	// Reconciles the desired state
+	// Example for an installation flow:
+	// 	1. Ensures the infra via AWS Cloudformation
+	// 	2. Ensures ROKS Cluster installation on existing infra
+	// 	3. Ensures ManagedFusionAgent Operator on ROKS Cluster
+	// 	4. Ensures presence of config for MFAO monitoring stack
+	// 	5. Ensures Data Foundation Config is present on ROKS Cluster
+	return forman.Result{}, nil
```

Comment:
As far as I am concerned this should be the content of `r.Reconcile`. Why do we need this second method for?

Reply:
done.